src/tests/Interop/COM/NETClients/Primitives/Program.cs

```diff
@@ -20,22 +20,51 @@ static int Main(string[] doNotUse)
 
             try
             {
+                Console.WriteLine(
+@"=========================
+Testing COM object interop.
+=========================");
                 RunTests();
-                Console.WriteLine("Testing COM object lifetime control methods.");
+            }
+            catch (Exception e)
+            {
+                Console.WriteLine($"Test object interop failure: {e}");
+                return 101;
+            }
+
+            try
+            {
+                Console.WriteLine(
+@"
+=========================
+Testing COM object lifetime control methods.
```

Comment:
Testing lifetime control seems weirdly tacked on to NETClientPrimitives? Are we just trying to avoid running the lifetime control tests if the first pass of just the object interop tests fails?

Reply:
I agree. Ideally we would move these to a newer more narrower test. For right now I just wanted to make sure it was no longer failing. Would you prefer to I simply rewrite the test now?

Reply:
I'd prefer to just split it now.